### DISCUSSION TITLE
in-built self-updating system mechanism

### DISCUSSION THREAD

**Discussion by ghost** [linuxserver/Heimdall#679] [Ideas]
can there be an update system in web ui??? 

idea:
- stable from https://github.com/linuxserver/Heimdall/tags
- dev from git master branch?

**Comment by funky4t** [comment]
I am on 2.2.3 an cannot find an update option inside the settings section. Is there an option anywhere? Or other way round, how can I update to the actual version. Sorry if bothering you with that, but I am not really into linux and could not find an option/terminal command on google. So I am the perfect dumb user.... If there is a command, simply put it under the settings section for users like me!
Thanks for Heimdall so far :)
